engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -193,31 +162,26 @@ public boolean pregenerateChunks() {
     }
 
     @Override
-    public void update(float delta) {
+    public void update(float deltaInMs) {
 
         PerformanceMonitor.startActivity("Update Tick");
```

Comment:
Another long-shot thought on the issue of multiple return locations:  python has a construct in the form:

    with(lock) {
   }

which allows to automatically release a lock whenever the code within the block returns or generates an exception. Is there any such thing in Java and could the PerformanceMonitor be made to behave similarly?